apps/sv/frontend/src/__tests__/governance/forms/grant-revoke-featured-app-form.test.tsx

```diff
@@ -247,19 +285,19 @@ describe('Revoke Featured App Form', () => {
     // completing the form should reenable the submit button
     const summaryInput = screen.getByTestId('revoke-featured-app-summary');
     expect(summaryInput).toBeDefined();
-    user.type(summaryInput, 'Summary of the proposal');
+    await user.type(summaryInput, 'Summary of the proposal');
```

Comment:
Learned over the weekend that these need to be awaited to avoid flakey tests so I'm adding them as I go along. Still strange to me that typing into a field isn't a synchronous thing